controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -945,6 +948,72 @@ bool master_thread::handle_cmdu_1905_channel_selection_response(Socket *sd,
 
     } while (cmdu_rx.getNextTlvType() != int(ieee1905_1::eTlvType::TLV_END_OF_MESSAGE));
 
+    return true;
+}
+
+bool master_thread::handle_cmdu_1905_link_metric_responce(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_rx)
+{
+
+    auto mid = cmdu_rx.getMessageId();
+    LOG(INFO) << "Received LINK_METRIC_RESPONSE_MESSAGE, mid=" << std::dec << int(mid);
+
+    // build metric responce message
+    if (!cmdu_tx.create(0, ieee1905_1::eMessageType::LINK_METRIC_RESPONSE_MESSAGE)) {
```

Comment:
I guess this is a copy-paste mistake? You don't need to construct a response here.